pkg/util/buildutil/crdb_test_off.go

```diff
@@ -19,3 +19,34 @@ package buildutil
 // metamorphic-style perturbations that will not affect test results but will
 // exercise different parts of the code.
 const CrdbTestBuild = false
+
+// TestingInt64 is an empty struct that can be used as a `gogoproto.casttype` in
+// proto messages. It uses no space. When the crdb_test build tag is set, this
+// type is instead represented by a RealTestingInt64.
+type TestingInt64 struct{}
```

Comment:
nit: maybe we should just move this stuff into `testing_int64` and gate it on `crdb_test`?